From abe67ee2e7f7688075983a5c52d254a794ace971 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Mon, 14 May 2018 10:07:57 +0800
Subject: [PATCH 019/129] media: intel-ipu4: css:
 scci_IPU4_master_20180511_2200 fw release

Message for Open Source:
New firmware css release.

Message for Internal:
[Feature]New firmware css release.
[Root Cause/Changes]
isys & psys package lib2600b0 for commit-id 9dd8b9e
FW_Manifest_ID: 6f1fb92662064046bb3c65e78ac570bfdabac0de
FW_Build_Branch: IPU4_master
FW_Build_Topic: auto_merge_queue
library version: commit-id 9dd8b9e
FW Bundle patch number of this release :  328262
FW tag : scci_IPU4_master_20180511_2200

Change-Id: I48b4b330b9052aca566dce85f3876e3d1df19292
Depends-on: I5f998f5704de9737161720bd123194141d668df4
Depends-on: I6b974ecca870998dfbd8929505b6c29fda8b0e3a
Depends-on: Iffe2ca0aa9666b0c5c88a66cb7e8bdf6a1b1c79c
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index eff260d..9ec820e 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180509
+#define IA_CSS_FW_PKG_RELEASE  0x20180511
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index eff260d..9ec820e 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180509
+#define IA_CSS_FW_PKG_RELEASE  0x20180511
-- 
1.9.1

